StackExRedis/NHibernate.Caches.StackExRedis/FastRegionStrategy.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using StackExchange.Redis;
+
+namespace NHibernate.Caches.StackExRedis
+{
+	/// <summary>
+	/// A region strategy that have very simple read/write operations but does not support
+	/// <see cref="Clear"/> operation.
+	/// </summary>
+	public partial class FastRegionStrategy : AbstractRegionStrategy
+	{
+		private const string SlidingGetLuaScript = @"
```

Comment:
While these scripts are quite short, they are also potentially used a LOT. It may be worthwhile to Load them into Redis, instead of sending the whole scripts every time.

Reply:
About the scripts not being loaded I also though that we have to manually load them based on [the documentation](https://stackexchange.github.io/StackExchange.Redis/Scripting) and I've made a `IScriptEvaluator` interface and implement it by loading the scripts to the server. After doing some tests and looking with the [Redis monitor](https://redis.io/commands/monitor) what commands are executed, I realized that even if we don't manually load the scripts on the server, `StackExchange.Redis` does this for us automatically (the first time it loads the script via `LOAD SCRIPT` and afterwards the `EVALSHA` is called). That is why I removed the `IScriptEvaluator` prior sending the PR to avoid unnecessary complexity.

Reply:
Wow! I had no idea. All good then.

Reply:
The lua scripts are now moved to embedded resources.